<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_dev">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Dev1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dev2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dev3</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Analog</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_start_read">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>49</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start read</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ao">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ao0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ao1</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_write">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>49</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_ao">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>49</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>49</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_calibre">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Read from :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Write to :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ai">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ai0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ai7</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibre :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tension :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop_read">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>49</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop read</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="PlotWidget" name="read_graph" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_read">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear graph</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="write_graph" native="true">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>80</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_test_sin">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Test sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_write">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>210</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop_sin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>210</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop sin</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_freq">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>210</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>212</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>f (Hz) :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>f_ech (Hz) :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_write_freq_ech">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800000</number>
     </property>
     <property name="value">
      <number>250000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_read_freq_ech">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>49</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>250000</number>
     </property>
     <property name="value">
      <number>250000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>f_ech (Hz) :</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
